src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -7,8 +7,9 @@
 // Public UI styles
 @import 'src/legacy/ui/public/index';
 
-// vis_type_vislib UI styles
-@import 'src/legacy/core_plugins/vis_type_vislib/public/index';
+// vis_type_vislib UI styles are imported here for running karma Browser tests
+// should be somehow included through the "vis_type_vislib" plugin initialization
+@import '../../../../plugins/vis_type_vislib/public/index';
```

Comment:
Since you have this imported here and in the plugin, doesn't it duplicate the compiled styles?

Reply:
Yes, it's duplicated temporary. We'll get rid of this legacy compilation for public code and will only use this for testing.

Reply:
@sulemanof For now we can comment out the import in the plugin because the chart will always render in the legacy platform. When cutting over visualize and dashboard to the new platform we can revisit this again.

Reply:
Thanks @flash1293 That's what I'd suggest as well. We really want to avoid duplicate styles

Reply:
@sulemanof I found a workaround we can use here. The `tests_bundle` is stripped out for the production build, but it's loaded in karma tests. We can put the "duplicated" styles there, so we don't pollute anything _and_ won't have problems when doing the cutover.

Reply:
@flash1293 thanks for your effort! I think it's the best solution! 
Merged into current branch.

Reply:
@cchaos are you OK with the changes?